src/java/org/apache/cassandra/db/compaction/CompactionTask.java

```diff
@@ -428,4 +403,115 @@ public static long getMaxDataAge(Collection<SSTableReader> sstables)
         }
         return max;
     }
+
+    private void debugLogCompactionSummaryInfo(
+    UUID taskId,
```

Comment:
Indentation is wrong both here and in the `logger.debug` statement below.

Reply:
Fixed in 142f9b47a8e7fa4c142da836b23df2ae66cad379.